src/harness/util.py

```diff
@@ -405,9 +405,9 @@ def filterChannelsByFrequencyRange(channels, freq_range):
   return [
```

Comment:
I don't understand where was the problem with this function. Could you give a clear example?
In my opinion, the new version does not give the expected result (and the SIQ3 test now fails...)

This function was a helper function only used by SIQ_3 testcase, in order to facilitate the different checks used in this testcase. 
Maybe the name of the function is not well chosen, and we could modify it to something like returnChannelsWithinFrequencyRange 

The objective of the function is that, given a list of channels (5 or 10 MHz), and given a frequency range (f_min, f_max), returns the list of channels that are included in the frequency range.

Let's take a simple example where the initial input channel list contains only 1 channel: 
3620000000 - 3630000000
Let' s take the frequency range as:
1) 3630000000 - 3650000000

The expected result is [None] since the channel is not within the frequency range.
With the new version, the result is the initial channel 3620000000 - 3630000000.
Clearly not the expected result.

By the way, this test is now failing after this change.

Reply:
see [Issue 654](https://github.com/Wireless-Innovation-Forum/Spectrum-Access-System/issues/654).
We will make a change to address the bug you have raised.